examples/experimental/riscv64.yaml

```diff
@@ -1,22 +1,15 @@
-# This template requires Lima v0.11.0 or later.
+# This template requires Lima v1.0 or later, with QEMU v9.1 or later.
```

Comment:
Since you have to resolve merge conflicts anyways, you should also make use of the new keys:

```yaml
minimumLimaVersion: 1.0.0
vmOpts:
  qemu:
    minimumVersion: 9.1.0
```

Reply:
done